### ISSUE TITLE
Custom other options not being restored in the forms on a failed validation

### ISSUE THREAD

**Issue by luisgreg99** [osulp/Scholars-Archive#1362]
Custom 'other' options (`degree_level_other`, `degree_name_other`, `degree_grantors_other`, etc) are not being persisted/restored on edit/new forms when a validation fails.

**Comment by luisgreg99** [comment]
This one has been addressed in this commit https://github.com/osulp/Scholars-Archive/pull/1330/commits/626c379ee7e24bd0c04faea2e172274831eec650

**Comment by simholt** [comment]
**QA in Firefox, Chrome, & Safari** 
So this works but I noticed a couple things. @vantuyls thoughts pls. Is this good enough for now?

1. The typed term has to exactly match the term in the drop-down for the "this already exists" notification to pop up. I think this is okay -- it will cut down on user error, which is good.

2. In Firefox, the notification does not stick to the assigned field as you scroll. I think people can figure this out, but it doesn't look good. (It sticks in Chrome. In Safari, it disappears as soon as you start scrolling, which isn't great.)

![Screen_Shot_2019-05-15_at_9_31_37_AM](https://user-images.githubusercontent.com/33705514/57792851-d76f3c80-76f4-11e9-97d9-361f58c98c11.png)

**Comment by vantuyls** [comment]
i would prefer this stick. @luisgreg99 any thoughts as to why this isn't working as expected in Firefox or Safari?

**Comment by luisgreg99** [comment]
It's currently using the browser's default style and behavior, so it changes depending on the browser. I double checked and can confirm that it doesn't stick in Firefox and Safari. It looks like it's possible to customize the look and behavior so that it's consistent across browsers. I'm currently looking into this one in the documentation https://developer.mozilla.org/en-US/docs/Learn/HTML/Forms/Form_validation I think the last example might do it.

**Comment by luisgreg99** [comment]
@simholt @vantuyls I got it working without using the built-in behavior, so the validation message should stick to the input when scrolling. The only caveat with this new method is that the browser-defined balloon/pup-up would need to be replaced with a label to keep scrolling consistent across browsers. I tested this in Firefox, Safari, and Chrome. Here is a preview for Degree Field:
![custom-client-validation-1](https://user-images.githubusercontent.com/3486120/57871148-88410e80-77bd-11e9-8bf7-701c4f25749b.gif)
